config/pipelines.yml

```diff
@@ -180,10 +180,56 @@ default: &default
           type: :model
           value: request.sample_name
           populate_on_row_type: :sample
-  ont:
+  ont: 
     covid:
       pcr_tag_set_name: "OntWell96Samples"
-
+      message:
+        lims: *lims
+        key: oseq_flowcell
+        fields:
+          last_updated: *timestamp
+          id_flowcell_lims:
+            type: :model
+            value: library.flowcell.id
+          sample_uuid:
+            type: :model
+            value: external_id
+          study_uuid:
+            type: :string
+            value: "test study id"
+          experiment_name:
+            type: :model
+            value: library.flowcell.run.id
+          instrument_name:
+            type: :string
+            value: GridION # TODO - change once known
+          instrument_slot:
+            type: :model
+            value: library.flowcell.position
+          pipeline_id_lims:
+            type: :string
+            value: "test value" # TODO - change once known
+          tag_identifier:
+            type: :model
+            value: tags.first.id
+          tag_sequence:
+            type: :model
+            value: tags.first.oligo
+          tag_set_id_lims: 
+            type: :model
+            value: tags.first.tag_set.id
+          tag_set_name:
+            type: :model
+            value: tags.first.tag_set.name
+          tag2_identifier: &nil
```

Comment:
I think you can just do but not sure:
`You can just do tags.&last.&tag_set.&name`
There might be another way where we could use the children method but that might not work because of the tag and tag2 thing.

Reply:
Would tags&.last not just return the first tag?

Reply:
Yes you're right it would. tags[1] wouldn't work either. I think you can just return an empty string rather than adding extra logic. I will double check.